2-Authorization-I/1-call-graph/App/authRedirect.js

```diff
@@ -55,42 +111,106 @@ function signIn() {
 }
 
 function signOut() {
-
+    
     /**
      * You can pass a custom request object below. This will override the initial configuration. For more information, visit:
      * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/request-response-object.md#request
      */
 
     // Choose which account to logout from by passing a username.
+    const account = myMSALObj.getAccountByUsername(username);
     const logoutRequest = {
-        account: myMSALObj.getAccountByUsername(username)
+        account: account,
     };
 
-    myMSALObj.logout(logoutRequest);
+    clearStorage(account);
+    myMSALObj.logoutRedirect(logoutRequest);
```

Comment:
Can we double check if this actually selects which account to sign-out by default when navigating to AAD sign-out screen?

Reply:
We will need the login_hint for in the case of the redirect.